src/DDoSX/DomainPropertiesClient.php

```diff
@@ -0,0 +1,45 @@
+<?php
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\Client as BaseClient;
+use UKFast\SDK\DDoSX\Entities\DomainProperty;
+use UKFast\SDK\SelfResponse;
+
+class DomainPropertiesClient extends BaseClient
+{
+    protected $basePath = 'ddosx/';
+
+    const DOMAIN_PROPERTIES_MAP = [];
+
+    /**
+     * Update a Domain's property
+     * @param $domainId
+     * @param $propertyId
+     * @param $data
+     * @return SelfResponse
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function update($domainId, $propertyId, $data)
+    {
+        $requestBody = [
```

Comment:
we need to `json_encode` this else I get the error `Passing in the "body" request option as an array to send a POST request has been deprecated. ` 

```suggestion
        $requestBody = json_encode(['value' => $data->value]);
```